src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -682,6 +694,121 @@ private static bool TryParseExactX(ReadOnlySpan<char> guidString, ref GuidResult
             return true;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is the `AggressiveInlining` helping perf here? It's a fairly large and branchy method.

Reply:
Yes, without inlining TryParseExactD takes 82ns (vs. 65).